app/src/main/java/com/github/sdp/mediato/data/UserDatabase.java

```diff
@@ -296,5 +296,33 @@ public static CompletableFuture<List<String>> getNearbyUsers(String username, do
                 });
         return future;
     }
+
+    /**
+     * Gets all the users
+     * @param username of the reference user
+     * @return a completable future with a list of users containing all the user other than the
+     *          user itself.
+     */
+    public static CompletableFuture<List<User>> getAllUser(String username) {

```

Comment:
Why do you need to get all the users instead of just their usernames and then retrieving the specific user if needed ?

Reply:
Because for the moment I use a list of user inside my code. But could be change in a future refactoring of the code.